### PR TITLE
Set and reset Locale to pass test for non-english contributor.

### LINKED ISSUE #2621: Test was not passed when Default language is not English
## Expected Behavior
Expect default forks repository tests will pass but It doesn't, because of default language setting.

## Observed Behavior
My current language is Korean(한국어), so the throws is also written in Korean.
But set expect was written in English. See below image.
bug

## Steps to Reproduce
Set non-english and run test.

## Your Environment

* Version of ktlint used: 1.2.1
* Relevant parts of the `.editorconfig` settings
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): Gradle 8.7
* Version of Gradle used (if applicable): Gradle 8.7
* Operating System and version: Window 11

### PR DESCRIPTION
**Pull request by Jaehwa-Noh**
## Description
When I build code on local machine, some tests failed because of current language setting.
See this issue.
Fixes #2621 

Then I temporary set the default language from Default to English.
 
## Checklist

Before submitting the PR, please check following (checks which are not relevant may be ignored):
- [x] Commit message are well written. In addition to a short title, the commit message also explain why a change is made.
- [x] At least one commit message contains a reference `Closes #<xxx>` or `Fixes #<xxx>` (replace`<xxx>` with issue number)
- [x] Tests are added
- [x] KtLint format has been applied on source code itself and violations are fixed
- [x] PR title is short and clear (it is used as description in the release changelog)
- [x] PR description added (background information)

[Documentation](https://pinterest.github.io/ktlint/) is updated. See [difference between snapshot and release documentation](https://github.com/pinterest/ktlint/tree/master/documentation)
- [x] [Snapshot documentation](https://github.com/pinterest/ktlint/tree/master/documentation/snapshot) in case documentation is to be released together with a code change
- [x] [Release documentation](https://github.com/pinterest/ktlint/tree/master/documentation/release-latest) in case documentation is related to a released version of ktlint and has to be published as soon as the change is merged to master

### CODE REVIEW

File: ktlint-cli-reporter-baseline/src/test/kotlin/com/pinterest/ktlint/cli/reporter/baseline/BaselineTest.kt [hunk 1]
```diff
@@ -148,6 +155,10 @@ class BaselineTest {
 
             assertThat(logCaptor.errorLogs).contains("Unable to parse baseline file: $path")
         }
+
+        private fun setLocaleToEnglish() = Locale.setDefault(Locale.ENGLISH)
+
+        private fun resetLocale(default: Locale) = Locale.setDefault(default)
```
Comment (paul-dingemans) [inline ktlint-cli-reporter-baseline/src/test/kotlin/com/pinterest/ktlint/cli/reporter/baseline/BaselineTest.kt:161]: When code is moved to BeforeEach and AfterEach, it is not necessary those helper methods. It is pretty obvious what is done.

**Review by paul-dingemans — CHANGES_REQUESTED** [review]
Tnx for reporting an fixing. Please address the remarks.

**Comment by Jaehwa-Noh** [comment]
Thank you for reviewing.
I'd changed what you addressed.

**Comment by paul-dingemans** [comment]
Tnx for your contribution

**Pull request merged**